### DISCUSSION TITLE
在 Android 设备直接使用 v2ray-core 的场景下，如何共享热点

### DISCUSSION THREAD

**Discussion by Aniark** [Mygod/VPNHotspot#349] [Q&A] [answered]
如题，[选择 v2ray-core 而不是套壳 app 是因为这样设备不会发热和增加耗电量](https://github.com/v2fly/v2ray-step-by-step/blob/transifex/zh_CN/app/android_without_root_use_v2ray.md#其他)，美中不足的是热点共享成了问题

**Comment by Mygod** [comment, marked as answer]
If you are configuring it via APN proxy then system hotspot should work already?

**Reply by Aniark** [reply]
但目前却未能如期让其它设备共享我的 v2ray
wlan 热点与 usb 网络共享均不能

一些并未起作用的尝试：
1，试用所有的 ip 掩蔽模式
2，试用所有的网络状态监听模式
3，逐个开关所有其它设置选项

**Reply by Aniark** [reply]
我的 v2ray config.json 

{
  "inbounds": [
    {
      "tag": "in",
      "port": 6,
      "protocol": "http",
      "sniffing": {
        "enabled": true,
        "destOverride": [
            "http",
            "tls"
        ]
      }
    }
  ],
  "outbounds": [
        {
            "tag": "proxy",
            "bla bla bla"
        },
        {
            "tag": "direct",
            "protocol": "freedom"
        }
  ],
  "routing": {
  	"rules":[
      {
      	"bla bla bla"
      }
    ]
  }
}

apn 亦已配置对应端口的代理

**Reply by Mygod** [reply]
This app does not support non-VPN solutions.
